src/ray/common/ray_config_def.h

```diff
@@ -254,7 +254,7 @@ RAY_CONFIG(uint32_t, cancellation_retry_ms, 2000)
 RAY_CONFIG(int64_t, ping_gcs_rpc_server_interval_milliseconds, 1000)
 
 /// Maximum number of times to retry ping gcs rpc server when gcs server restarts.
-RAY_CONFIG(int32_t, ping_gcs_rpc_server_max_retries, 1)
+RAY_CONFIG(int32_t, ping_gcs_rpc_server_max_retries, 600)
```

Comment:
1 vs 600 sounds really different. Can you also comment how long time it will keep retrying?

Reply:
I didn't test how many times it retries. I just found that 0.8.7 uses 600 retries.